apkid/__init__.py

```diff
@@ -34,7 +34,7 @@
 
 import argparse
 
-import apkid
+from . import apkid
```

Comment:
Curious, why is this needed or an improvement?

Reply:
Agreed, it's likely for that, however I don't see the need for it I guess without an explanation as to why this is better?